script/virustotal_scan.py

```diff
@@ -46,20 +47,32 @@ def parse_response(response):
         return SCAN_ERROR
 
     # print(response)
-    total = len(response['data']['attributes']['last_analysis_results'])
+
+    results = response['data']['attributes']['last_analysis_results']
+
+    total = len(results)
     malicious = response['data']['attributes']['last_analysis_stats']['malicious']
     permalink = response['data']['links']['self'].replace(
         "api/v3/files", "gui/file")
 
     if malicious == 0:
         return SCAN_CLEAN
+    elif malicious == 1:
+        LOGGER.warning(
+            f"Known false-positive threats detected (Antiy-AVL or Zillya): {permalink}")
+        if (results['Antiy-AVL']['result'] is not None or results['Zillya']['result'] is not None):
+            return SCAN_FALSE_POSITIVES_FOUND
+    elif malicious == 2:
+        LOGGER.warning(
+            f"Known false-positive threats detected (Antiy-AVL and Zillya): {permalink}")
+        if (results['Antiy-AVL']['result'] is not None and results['Zillya']['result'] is not None):
+            return SCAN_FALSE_POSITIVES_FOUND
     else:
         LOGGER.warning(f"Threats detected: {permalink}")
         return SCAN_THREATS_FOUND
```

Comment:
```suggestion
    else:
        ignore = ['Antiy-AVL', 'Zillya']
        ignored_results = [n for n in ignore if results[n]['result'] is not None]
        if malicious == len(ignored_results):
            f"Known false-positive threats detected ({', '.join(ignored_results)}): {permalink}")
            return SCAN_FALSE_POSITIVES_FOUND
        LOGGER.warning(f"Threats detected: {permalink}")
        return SCAN_THREATS_FOUND
```

Currently this will return a None value as the `else` block will not be hit if we have 1 or 2 AVs marking as malicious and they are not on our ignore list.